src/Components/GameScreenComponents/Column.tsx

```diff
@@ -1,24 +1,59 @@
 import React from "react"
-import "../../styles/GameScreen.css"
-import { numOfRows } from "../../Constants";
+import "../GameScreen.css"
+import {numOfRows} from "../../Constants";
+import checkGameStatus, {gameResult, gameStatus} from "./WinningLogic";
 
 interface columnInfo{
     columnLetter: string
+    columnNumber: number
+    gameColumn: number[]
+    gameBoard:number[][]
+    playerColors: {Player1Color:string, Player2Color:string}
+    playerCoordinates: {Column:number, Row:number}
+    setPlayerCoordinates: React.Dispatch<React.SetStateAction<{Column: number, Row: number}>>
+    gameStatus: {result: gameResult, playerNumber: number, gameBoard: number[][]}
+    setGameStatus: React.Dispatch<React.SetStateAction<{result: gameResult, playerNumber: number, gameBoard: number[][]}>>
+    playerScores: {Player1Score: number, Player2Score: number}
+    setPlayerScores:  React.Dispatch<React.SetStateAction<{Player1Score: number, Player2Score: number}>>
 }
 
+/*One of the main components that updates the state every time a player clicks on a column*/
 function Column(props: columnInfo){
-    const columns = [];
+
+    const column = [];
+    let endGameResult:gameStatus;
```

Comment:
I'd recommend defining `endGameResult` inside handleColumnClick (inside props.setGameStatus) as it looks like that's the only place it's used